src/seedu/addressbook/AddressBook.java

```diff
@@ -498,12 +497,28 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
         final ArrayList<String[]> matchedPersons = new ArrayList<>();
         for (String[] person : getAllPersonsInAddressBook()) {
             final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person)));
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            if (!Collections.disjoint(makeLowerCase(wordsInName), makeLowerCase(keywords))) {
                 matchedPersons.add(person);
             }
         }
         return matchedPersons;
     }
+    
+    /**
+     * Makes set of Strings lowercase
+     */
```

Comment:
Normally, you should give a description of the `@param` or the `@return` value)

When you modify the behavior, remember to update documentation and tests as well.